prusti-encoder/src/encoders/type/domain.rs

```diff
@@ -156,10 +210,17 @@ impl TaskEncoder for DomainEnc {
         vir::with_vcx(|vcx| {
             let mut builder = PureTypeCommon::new(vcx);
 
-            if matches!(task_key.kind(), TyKind::Param(_)) {
-                let (specifics, builder) = super::kinds::param::domain(*task_key, deps, builder)?;
-                return Ok((PureTypeCommon::build(builder), specifics));
-            }
+            // if matches!(task_key.kind(), TyKind::Param(_)) {
+            //     let (oref, specifics) = super::kinds::param::domain(*task_key, deps, builder)?;
+            //     deps.emit_output_ref(
+            //         *task_key,
+            //         oref.clone(),
+            //     )?;
+            //     return Ok((PureTypeCommon::build(builder), DomainEncOutput {
+            //         inner: oref,
+            //         specifics,
+            //     }));
+            // }
```

Comment:
(remove comment)

So does `TyKind::Param` get intercepted elsewhere now? Or is it encoded as an opaque domain? I also don't see how its `emit_output_ref` gets called: it's not here, and you also removed that call from `...::types::kinds::param::domain`.